messages/snapshot.md

```diff
@@ -4,4 +4,4 @@ No snapshot found with the given name or id: %s
 
 # snapshotNotEnabled
 
-Org snapshots aren’t enabled for your Dev Hub.
+Org Snapshots isn't enabled for your Dev Hub.
```

Comment:
```suggestion
Scratch Org Snapshots isn't enabled for your Dev Hub.
```

Reply:
I added "scratch" because this doc says that the toggle to enable snapshots is called "Enable Scratch Org Snapshots", and it's good to try to use the same verbiage as in the UI: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_snapshots_enable_snapshots_devhub.htm.

Reply:
I'll bring it up to the our doc writer.

Reply:
Ok we're good with the suggestion, thanks!